Sources/WinUIBackend/WinUIBackend.swift

```diff
@@ -0,0 +1,447 @@
+import SwiftCrossUI
+import UWP
+import WinAppSDK
+import WindowsFoundation
+import WinUI
+
+class WinRTApplication: SwiftApplication {
+    static var callback: () -> Void = {}
+
+    required init() {
+        super.init()
+    }
+
+    override public func onLaunched(_: WinUI.LaunchActivatedEventArgs) {
+        Self.callback()
+    }
+}
+
+public struct WinUIBackend: AppBackend {
+    public typealias Window = WinUI.Window
+    public typealias Widget = WinUI.UIElement
+
+    private class InternalState {
+        var buttonClickActions: [ObjectIdentifier: () -> Void] = [:]
+        var toggleClickActions: [ObjectIdentifier: (Bool) -> Void] = [:]
+        var switchClickActions: [ObjectIdentifier: (Bool) -> Void] = [:]
+        var sliderChangeActions: [ObjectIdentifier: (Double) -> Void] = [:]
+        var pickerChangeActions: [ObjectIdentifier: (Int?) -> Void] = [:]
+        var textFieldChangeActions: [ObjectIdentifier: (String) -> Void] = [:]
+    }
+
+    private var internalState: InternalState
+
+    public init(appIdentifier _: String) {
+        internalState = InternalState()
+    }
+
+    public func runMainLoop(_ callback: @escaping () -> Void) {
+        WinRTApplication.callback = callback
+        WinRTApplication.main()
+    }
+
+    public func createWindow(withDefaultSize size: SwiftCrossUI.Size?) -> Window {
+        let window = Window()
+
+        if let size {
+            try! window.appWindow.resizeClient(SizeInt32(width: Int32(size.width), height: Int32(size.height)))
```

Comment:
Partly to please the linter, and partly for practical reasons, it could be a good idea to instead wrap the try in a `do catch` and then `fatalError` in the `catch` so that we can provide an error message (`try!` often prints some pretty useless stuff in my experience, but it differs per platform)

Reply:
Ignore that, Windows needs lots of `try!`, so I'll probably just remove that linting rule (as I did for `as!`)